package.json

```diff
@@ -42,6 +42,7 @@
 	},
 	"devDependencies": {
 		"@babel/plugin-transform-modules-commonjs": "^7.15.4",
+		"@types/jsonwebtoken": "^8.5.5",
```

Comment:
why did you installed this?

Reply:
It gives me error and the id verification doesn't work. So I quick fix it through vs and it installed automatically.